PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/CacheManager.java

```diff
@@ -0,0 +1,72 @@
+/*
+ *    Copyright 2020-2021 Prebid.org, Inc.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package org.prebid.mobile;
+
+import android.os.Handler;
+import android.support.annotation.VisibleForTesting;
+import android.text.TextUtils;
+
+import java.util.HashMap;
+import java.util.UUID;
+
+public class CacheManager {
+    private static final int NATIVE_AD_EXPIRY_TIMEOUT = 300000;
+    private static HashMap<String, String> savedValues = new HashMap<>();
+    private static HashMap<String, CacheExpiryListener> cacheExpiryListenerMap = new HashMap<>();
+    private static Handler handler = new Handler();
+
+    public static String save(String content) {
+        if (!TextUtils.isEmpty(content)) {
+            final String cacheId = "Prebid_" + UUID.randomUUID().toString();
+            savedValues.put(cacheId, content);
+            handler.postDelayed(new Runnable() {
+                @Override
+                public void run() {
+                    if (cacheExpiryListenerMap.containsKey(cacheId)) {
+                        cacheExpiryListenerMap.remove(cacheId).onCacheExpired();
+                    }
+                    savedValues.remove(cacheId);
+                }
+            }, NATIVE_AD_EXPIRY_TIMEOUT);
+            return cacheId;
+        } else {
+            return null;
+        }
+    }
```

Comment:
@anwzhang 

As I see cache is cleared after 5 minutes (NATIVE_AD_EXPIRY_TIMEOUT, 300000).
Maybe we should rely on `seatbid.bid.exp` ?

Reply:
Is the expiration value provided by seatbid?

Reply:
I see. Rubicon PBS contains `exp` in `seatbid.bid` object. 
In this case we can use a fallback logic
@bszekely1, what do you think ?

Reply:
The idea behind using the `seatbid.bid.exp` is this is provided by PBS to know how long a bid can stay valid for. If this value is set, use this as the bid cache expiration, else, use a default value.

Reply:
Done with the changes. Please review if that looks fine.